# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/win/src/sandboxtarget/moz.build
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 3
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/win/src/sandboxtarget/moz.build
--- a/security/sandbox/win/src/sandboxtarget/moz.build	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/win/src/sandboxtarget/moz.build	Wed Jun 12 11:10:48 2019 +0100
@@ -17,4 +17,6 @@
     '/security/sandbox/chromium-shim',
 ]
 
+DEFINES['UNICODE'] = True
+
 FINAL_LIBRARY = 'xul'
